libnavigation-core/src/main/java/com/mapbox/navigation/core/routerefresh/RouteRefreshExtra.kt

```diff
@@ -0,0 +1,42 @@
+package com.mapbox.navigation.core.routerefresh
+
+import androidx.annotation.StringDef
+import com.mapbox.api.directions.v5.models.Incident
+import com.mapbox.api.directions.v5.models.LegAnnotation
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+
+/**
+ * Extra data of route refresh
+ */
+@ExperimentalPreviewMapboxNavigationAPI
+object RouteRefreshExtra {
+
+    /**
+     * The state becomes [REFRESH_STATE_STARTED] when route refresh round is started.
+     */
+    const val REFRESH_STATE_STARTED = "STARTED"
+
+    /**
+     * The state becomes [REFRESH_STATE_FINISHED_SUCCESS] when the route is successfully refreshed.
+     */
+    const val REFRESH_STATE_FINISHED_SUCCESS = "FINISHED_SUCCESS"
+
+    /**
+     * The state becomes [REFRESH_STATE_FINISHED_FAILED] when route refresh is failed and the route
+     * is cleaned up of invalid data (if it's not yet):
+     * - congestions annotations (see [LegAnnotation.congestion] and [LegAnnotation.congestionNumeric]);
+     * - incidents whose `endTime` is less than current time (see [Incident.endTime]).
```

Comment:
```suggestion
     * The state becomes [REFRESH_STATE_FINISHED_FAILED] when a route refresh failed and the route
     * is cleaned up of expired data, see [RouteRefreshOptions] for details.
```
Let's point to a single source of truth.